docs/DeveloperGuide.md

```diff
@@ -567,64 +567,211 @@ testers are expected to do more *exploratory* testing.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+### Deleting a Person
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+Prerequisite: There is at least 1 person in the Person List Panel.
 
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+`deletePerson 1`
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+Expected Output in the Person List Panel: The first person in the list is deleted.
 
-1. _{ more test cases …​ }_
+Expected Output in the Command Output Panel: Deleted person message is shown along with the person's details.
 
-### Deleting a transaction
+`deletePerson 0`
 
-1. Deleting a transaction while all transactions are being shown
+Expected Output in the Command Output Panel: Error message stating that there is an invalid command format.
+### Editing a Person's details
 
-    1. Prerequisites: There already exist some transactions in the application.
+Prerequisite: There is at least 1 person in the Person List Panel.
 
-    1. Test case: `deleteTransaction 1`<br>
-       Expected: First transaction is deleted from the list. Details of the deleted transaction shown in the status message. Timestamp in the status bar is updated.
+`editPerson 1 n=Ryan`
 
-    1. Test case: `delete 0`<br>
-       Expected: No transaction is deleted. Error details shown in the status message. Status bar remains the same.
+Expected Output in the Person List Panel: The first person in the list has their name changed to "Ryan", and retains
+ the rest of their details.
 
-    1. Other incorrect delete commands to try: `deleteTransaction`, `deleteTransaction x`, `...` (where x is larger than the list size)<br>
-       Expected: Similar to previous.
+Expected Output in the Command Output Panel: Edit person message is shown along with the person's details.
 
-1. _{ more test cases …​ }_
+`editPerson 0`
 
-### Saving data
+Expected Output in the Command Output Panel: Error message stating that there is an invalid command format.
 
-1. Dealing with missing/corrupted data files
+### Listing multiple people with matching names
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+Prerequisite: There is only 1 person stored in Spend N Split. Their name is
+ "Roy Tan".
 
-1. _{ more test cases …​ }_
+`listPerson Roy`
+
+Expected Output in the Person List Panel: The person Roy Tan is displayed.
+
+Expected Output in the Command Output Panel: List person message is shown and states "1 person(s) listed".
+
+`listPerson Tan`
+
+Expected Output in the Person List Panel: Nothing is displayed.
```

Comment:
I think the expected behaviour is that Roy Tan is displayed

Reply:
fixed